packages/fx-core/resource/package.nls.json

```diff
@@ -929,6 +935,10 @@
   "driver.spfx.warn.noTenantAppCatalogFound": "No tenant app catalog found, retry: %s",
   "driver.spfx.error.failedToGetAppCatalog": "Cannot get app catalog site url after creation. You may need wait a few minutes and retry.",
   "driver.spfx.error.noValidAppCatelog": "There is no valid app catalog in your tenant. You can update the property 'createAppCatalogIfNotExist' in %s to true if you want Teams Toolkit to create it for you or you can create it by yourself.",
+  "driver.spfx.add.description": "add additional web part to SPFx project",
```

Comment:
Capitalize 'a'

Reply:
This is as designed by unified output of drivers, all driver description are not capitalized. e.g. https://github.com/OfficeDev/TeamsFx/blob/1aadc352afab2cd34052596bcb377e1fcf634c9a/packages/fx-core/resource/package.nls.json#L917